experiments/calibration/shared/observation_data_locations.jl

```diff
@@ -0,0 +1,41 @@
+era5_ds = Dataset(
+    joinpath(
+        ClimaLand.Artifacts.era5_surface_data2008_path(),
+        "era5_monthly_surface_fluxes_200801-200812.nc",
+    ),
+)
+
+# Make the ERA5 target
+ERA5_target = []
+close = (x, y) -> abs(x - y) < 5e-1
```

Comment:
This threshold should depend on the grid spacing. Maybe it should be find closest? What happens if many points are close enough by this metric?